src/pages/en/core-concepts/astro-components.md

```diff
@@ -105,11 +105,36 @@ const myFavoritePokemon = [/* ... */];
 <p class:list={["add", "dynamic", {classNames: true}]} />
 ```
 
-### JSX Expressions
+## JSX-like Expressions
 
-You can can define local JavaScript variables inside of the frontmatter component script within an Astro component. You can then inject these variables into the component's HTML template using JSX expressions!
+You can can define local JavaScript variables inside of the frontmatter component script within an Astro component. You can then inject these variables into the component's HTML template using JSX-like expressions!
 
-#### Variables
+### `.astro` vs `.jsx`
+
+Astro component syntax is a superset of HTML. It was designed to feel familiar to anyone with HTML or JSX experience.
+
+**If you know HTML, you already know enough to write your first Astro component.**
+
+| Feature                      | Astro | JSX  |
+| ---------------------------- | ----- | --------- |
+| File extension               | `.astro` | `.jsx` or `.tsx` |
+| User-Defined Components      | `<Capitalized>` | `<Capitalized>`  |
+| Expression Syntax            | `{}` | `{}` |
+| Spread Attributes            | `{...props}` | `{...props}` |
+| Boolean Attributes           | `autocomplete` === `autocomplete={true}` | `autocomplete` === `autocomplete={true}` |
+| Inline Functions             | `{items.map(item => <li>{item}</li>)}`  | `{items.map(item => <li>{item}</li>)}` |
+| Conditional Rendering             | `{condition &&  <p>text<p>}`  | `{condition &&  <p>text<p>}` |
+| IDE Support                  | [VS Code (incl. Open VSX), Nova](/en/editor-setup/) | Phenomenal |
+| Requires JS import           | No    | Yes, `jsxPragma` (`React` or `h`) must be in scope |
+| Fragments                    | Automatic top-level, `<Fragment>` or `<>` inside functions | Wrap with `<Fragment>` or `<>` |
+| Multiple frameworks per-file | Yes | No |
+| Modifying `<head>`           | Just use `<head>` in top-level pages | Per-framework (`<Head>`, `<svelte:head>`, etc) |
+| Comment Style                | `<!-- HTML -->` | `{/* JavaScript */}`  |
+| Special Characters           | `&nbsp;`  | `&nbsp;`  |
+| Attributes                   | `dash-case` | `camelCase`|
+
+
+### Variables
```

Comment:
Remove the table in favour of the more detailed but less overwhelming version suggested. Do we still want this “If you know HTML…” line somewhere though? Leaving it here for now but maybe even move right up to somewhere more prominent?

```suggestion
**If you know HTML, you already know enough to write your first Astro component.**

### Variables
```